### ISSUE TITLE
JuliaParBlock internals not being highlighted

### ISSUE THREAD

**Issue by rcnlee** [JuliaEditorSupport/julia-vim#68]
Hello,

I am trying to get semantic-highlight.vim to work properly with julia-vim.  variables inside brackets () or [] or {} seem to be picked up as JuliaParBlock as a whole and so the individual variable names don't get parsed or highlighted.  Please see screenshot here: 
![screenshot from 2016-06-22 22-12-29](https://cloud.githubusercontent.com/assets/7119868/16292803/8b678a98-38c9-11e6-9de3-56a302345cd3.png)

Is there any way to get the individual tokens picked out by semantic-highlight?

Thanks!

Ritchie

**Comment by carlobaldassi** [comment]
I just tried it and in fact I'm not sure how to fix this. I read the semantic-highlight documentation and did not find anything useful to solve the problem. I also tried to compare with other syntax files and I didn't find anything significantly different in how things are done in the julia-vim plugin.

I guess your best bet is filing an issue at semantic-highlight.vim and see if there is some kind of customization required (e.g. I saw that filetypes need to specify keywords lists, so something similar may be necessary here?)
